### PR TITLE
game setup detail

### LINKED ISSUE #1598: Request: on initial card selection screen, tell all players what game options are in effect
On initial card selection screen, it would be nice to tell all players what game options are in effect.

### PR DESCRIPTION
**Pull request by vsrisuknimit**
This PR replaces #2372 and closes #1598 .

It uses GameOptionsModel in GameHome and in Preferences.

image

image

image

### CODE REVIEW

**Comment by vsrisuknimit** [comment]
Added solo setup details

image

**Comment by kberg** [comment]
The purpose of the extra material on the bottom of the preferences is -- is it informative or options?

**Comment by vsrisuknimit** [comment]
@kberg Informative. See #1598 . Sometimes users don't know what is the game setup.

**Comment by kberg** [comment]
Gotcha. So, I question whether it belongs on the set-up panel.

I suggest a new Info panel which contains all that data, plus the keyboard shortcuts.

**Comment by kberg** [comment]
Imagine an (i) icon above the (?) icon, for instance.

**Comment by vsrisuknimit** [comment]
Added i button. Put hotkey mapping and game setup details there.

image

File: src/components/Preferences.ts [hunk 1]
```diff
@@ -188,6 +189,9 @@ export const Preferences = Vue.component('preferences', {
         return `${rulingPartyName}`;
       }
     },
+    getToolTip: function(tip :string): string {
+      return translateText(tip);
```
Comment (alrusdi) [inline src/components/Preferences.ts:193]: Ugh! I'm sorry, you can use $t instead of translateText and it is already imported.

Comment (vsrisuknimit) [reply src/components/Preferences.ts]: Fixed. Use $t now.

**Review by alrusdi — APPROVED** [review]

**Pull request merged**